recipes/nb_conda/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+fontconfig
```

Comment:
I am having a *deja vu* here (maybe you already explained this to me sorry for asking again :grimacing:) but why `conda-forge`'s `fontconfig` does not work for here?